### DISCUSSION TITLE
[Marketplace] Define API Documentation Path

### DISCUSSION THREAD

**Discussion by ghost** [mia-platform/community#322] [Q&A] [answered]
### Which section?

Marketplace

### Which is the version?

11.2.0

### Description

Hi all!

 The Console let me define the Path to be used by the swagger-aggregator to retrieve the documentation of the API defined in the microservices. 
From the CMS, how can I define a custom documentation path to be used to retrieve the Swagger documentation of a Template Marketplace item?

Thank you!

**Comment by pmarchini** [comment, marked as answer]
Hey @ivanpasquariello ,

In the service resource definition you can add a property `defaultDocumentationPath`

e.g. : 

![image](https://github.com/mia-platform/community/assets/49943249/08746728-f627-4c0d-a50f-84b4ffd80b5d)

![image](https://github.com/mia-platform/community/assets/49943249/a635f6da-00ba-4156-bf84-132b4088590b)

**Reply by ghost** [reply]
Thank you Pietro! For reference, the info was here in the docs 
[Add a new element to the Marketplace](https://docs.mia-platform.eu/docs/marketplace/add_to_marketplace/contributing_overview#common-to-microservice-components-plugins-templates-examples)
I'll leave it here for everyone. Thank you once again!
